test/helper/testdata.go

```diff
@@ -0,0 +1,30 @@
+package helper
```

Comment:
Thank you for all the test data. That is amazing. We may want or need to break that out by MAAS version, for now its good.